pykechain/models/widgets/widgets_manager.py

```diff
@@ -1449,6 +1451,157 @@ def add_service_card_widget(
             **kwargs
         )
         return widget
+
+    def add_dashboard_widget(
+        self,
+        title: Optional[Union[bool, Text]] = False,
+        parent_widget: Optional[Union[Widget, Text]] = None,
+        source_projects: [DashboardWidgetSourceScopes] = DashboardWidgetSourceScopes.CURRENT_PROJECT,
+        source_projects_tags: Optional[List] = None,
+        source_subprocess: Optional[List] = None,
+        source_selected_scopes: Optional[List] = None,
+        show_tasks: Optional[List[DashboardWidgetShowTasks]] = None,
+        show_projects: Optional[List[DashboardWidgetShowScopes]] = None,
+        no_background: Optional[bool] = False,
+        show_assignees: Optional[bool] = True,
+        show_assignees_table: Optional[bool] = True,
+        show_open_task_assignees: Optional[bool] = True,
+        show_open_vs_closed_tasks: Optional[bool] = True,
+        show_open_closed_tasks_assignees: Optional[bool] = True,
+        **kwargs
+    ) -> Widget:
+        """
+        Add a KE-chain Dashboard Widget to the WidgetManager and the activity.
+
+        The widget will be saved in KE-chain
+
+        :param title: A custom title for the card widget
+            * False (default): Card name
+            * String value: Custom title
+            * None: No title
+        :param parent_widget: (O) parent of the widget for Multicolumn and Multirow widget.
+        :type parent_widget: Widget or basestring or None
+        :param source_projects: The `Project(s)` to be used as source when displaying the Widget.
```

Comment:
do we want to call this projects? or Scopes.

In the schema there are these 'source' options:

```json
"source": {
  "type": "string",
  "enum": [
    "scopes",
    "selectedScopes",
    "taggedScopes",
    "project",  # inconsistent!
    "subProcess"
  ]
}
```
Maybe ambiguous as well, but predominantly we use scopes everywhere I would say in development and for endusers we call it Projects.

It is better to constrain ourselve to the use of `scope` in pykechain as much as we can.

Also 'defaults on `CURRENT_PROJECT` should be renamed to --> `CURRENT_SCOPE`